src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx

```diff
@@ -530,7 +530,7 @@ class CreateSchemeRegistrarProposal extends React.Component<IProps, IState> {
                         <div className={css.permissionCheckbox}>
                           <Field id="mintBurnReputation" type="checkbox" name="mintBurnReputation" disabled="disabled" checked="checked" />
                           <label htmlFor="mintBurnReputation">
-                            Mint or burn reputation
+                            Mint and burn reputation, send ETH and external &amp; native tokens
```

Comment:
it should be added to all scheme information tab
<img width="1034" alt="Screen Shot 2020-06-01 at 10 40 58" src="https://user-images.githubusercontent.com/5994398/83386861-69c3aa80-a3f4-11ea-82ff-18fc15d6ea61.png">

Reply:
Done.  Note:  https://github.com/daostack/alchemy/issues/1785